gulpfile.js

```diff
@@ -52,96 +89,38 @@ gulp.task('copy-dirs', function() {
 });
 
 gulp.task('less', function() {
-  return gulp.src(LESS_FILES)
-    .pipe(handleError())
+  return gulp.src('./less/**/*.less')
+    .pipe(sourcemaps.init())
     .pipe(less({
-      paths: [path.join(__dirname, 'less')]
+      paths: ['./']
     }))
+    .pipe(sourcemaps.write('./'))
     .pipe(gulp.dest('./dist'));
 });
 
-gulp.task('webpack', function() {
-  return gulp.src(webpackConfig.entry)
-    .pipe(webpack(webpackConfig))
-    .pipe(gulp.dest('./dist'));
-});
-
-gulp.task('test', BUILD_TASKS.concat([
-  'test-react-warnings'
-]), function() {
-  gutil.log(gutil.colors.green.bold("Yay, tests pass!"));
-});
+// gulp.task('webpack', webpackTask({sourcemaps: true}));
+// gulp.task('webpack-optimized', webpackTask({optimize: true}));
```

Comment:
ideally I want this back, but not sure why gulp is firing these tasks even though I didn't have it anywhere...